### DISCUSSION TITLE
@config with @import doesn't appear to work

### DISCUSSION THREAD

**Discussion by RyanRoberts** [tailwindlabs/tailwindcss#9988] [Help] [answered]
Is this not possible? https://media.yep.digital/2MS3HG - when I try using @config (multi-configs) to create a separate css file it only works if Tailwind CSS is imported with @tailwind not @import. The generated CSS file is simply empty.

**Comment by adamwathan** [comment, marked as answer]
Hey! So postcss-import is very strict and requires all import statements be first in a file as per the CSS spec. You can solve this by just putting your `@config` declaration after your imports.

**Reply by RyanRoberts** [reply]
Right, yes that makes sense now. Thank you.
